packages/core/content-manager/server/src/controllers/collection-types.ts

```diff
@@ -511,7 +511,8 @@ export default {
       .populateFromQuery(permissionQuery)
       .build();
 
-    const entity = await entityManager.findOne(id, model, { populate });
+    const { locale, status = 'draft' } = getDocumentDimensions(ctx.query);
+    const entity = await entityManager.findOne(id, model, { populate, locale, status });
```

Comment:
could you update the `api-tests/core/content-manager/api/number-of-draft-relations.test.api.js` tests to validate this endpoint works ok with publication state? (locale if possible too, but that might be complicated)